library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3302,6 +3302,14 @@
     <string name="device_manager_session_rename_edit_hint">Session name</string>
     <string name="device_manager_session_rename_description">Custom session names can help you recognize your devices more easily.</string>
     <string name="device_manager_session_rename_warning">Please be aware that session names are also visible to people you communicate with.</string>
+    <string name="device_manager_learn_more_sessions_inactive_title">Inactive sessions</string>
+    <string name="device_manager_learn_more_sessions_inactive">Inactive sessions are sessions you have not used in some time, but they continue to receive encryption keys.\n\nRemoving inactive sessions improves security and performance, and makes it easier for you to identify if a new session is suspicious.</string>
+    <string name="device_manager_learn_more_sessions_unverified_title">Unverified sessions</string>
+    <string name="device_manager_learn_more_sessions_unverified">Unverified sessions are sessions that have logged in with your credentials but not been cross-verified.\n\nYou should make especially certain that you recognise these sessions as they could represent an unauthorised use of your account.</string>
```

Comment:
Is there a difference between cross-signing and cross-verifying? I'm mostly asking for translation purposes, but could it not also be confusing to native speakers? :)